Automattic/woocommerce-payments
# Moving hook initialisation functions out of constructors

Fixes #7262 

#### Changes proposed in this Pull Request

Remove hook initialisation out of constructors for the `WCPay_Account`, `*_Onboarding_Service` and `*_Incentive_Service`.









#### Testing instructions





* Tests should pass.
* PHPCS should pass.
* Compare the changeset for the `WCPay_Account` class - I slightly altered the order of actions/filters to order them more logically, but we should double check to make sure nothing went wrong.
* Give a light smoke test to the admin area and ensure everything works as expected. Particularly, we should ensure a new onboarding works. Run the listener on the server with `npm run listen`, disconnect the currently connected account on the store, perform an onboarding, clear the cache. Ensure everything in the admin area looks good. Make a test purchase, and make sure it displays properly on the `Payments > Transactions` page.

-------------------


**Post merge**
